include/wil/com.h

```diff
@@ -27,2099 +27,2160 @@
 /// @cond
 namespace Microsoft
 {
-    namespace WRL
-    {
-        template <typename T>
-        class ComPtr;
-    }
+namespace WRL
+{
+    template <typename T>
+    class ComPtr;
 }
+} // namespace Microsoft

```

Comment:
I'm not sure how I feel about the comment on the closing brace of each namespace.  For short ones like this it is overkill.

Reply:
I've never personally liked it either, but it's relatively minor and easy to ignore that idk if it's worth diverging from the OS `.clang-format` file